lumicks/pylake/detail/imaging_mixins.py

```diff
@@ -148,6 +149,8 @@ def export_video(
             When specified, we export a video correlated to channel data
         vertical : bool, optional
             Render with the plots vertically aligned (default: True).
+        downsample_channel : bool, optional
+            Downsample the channel data over the frame (default: True).
```

Comment:
perhaps:
```suggestion
            Downsample the channel data over frame timestamp ranges (default: True).
```
(would be for the other docstrings too)